manticore/core/smtlib/constraints.py

```diff
@@ -115,7 +116,7 @@ def _get_sid(self) -> int:
         self._sid += 1
         return self._sid
 
-    def __get_related(self, related_to=None):
+    def related_to(self, related_to: Optional[Expression] = None):
```

Comment:
Does adding this still work?
```suggestion
    def related_to(self, related_to: Optional[Expression] = None) -> ConstraintSet:
```